src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -444,6 +444,10 @@ class DocBaseViewer extends BaseViewer {
 
         this.pdfViewer.currentPageNumber = parsedPageNumber;
         this.cachePage(this.pdfViewer.currentPageNumber);
+
+        if (this.thumbnailsSidebar) {
+            this.thumbnailsSidebar.setCurrentPage(parsedPageNumber);
```

Comment:
Why do we need to do this here? The data goes from sidebar -> doc base, so why are we using doc base to report back?

Reply:
This is for the case where the page controls are used directly to change page (up, down, text input) in which case I need to communicate back to the thumbnails sidebar to also update the current page